content/pt-br/docs/concepts/configuration/windows-resource-management.md

```diff
@@ -0,0 +1,75 @@
+
+Em contraste, o Windows usa um [_objetos de trabalho_](https://docs.microsoft.com/windows/win32/procthread/job-objects) por contêiner com um filtro de namespace do sistema
+para conter todos os processos em um contêiner e fornecer isolamento lógico ao hospedar.
+(Os objetos de trabalho são um mecanismo de isolamento de processo do Windows e são diferentes dos
+que o Kubernetes chama de {{< glossary_tooltip term_id="job" text="Job" >}}).
+
+Não há como executar um contêiner do Windows sem a filtragem de namespace. 
+Isso significa que os privilégios do sistema não podem ser declarados no contexto do host e, 
+portanto, os contêineres privilegiados não estão disponíveis no Windows.
+Os contêineres não podem assumir uma identidade do host porque o Gerente de conta de segurança (SAM) é separado.
+
+## Gerenciamento de memória {#resource-management-memory}
+
+O Windows não possui um eliminador de processo de falta de memória como o Linux. 
+O Windows sempre trata todas as alocações de memória do modo de usuário como 
+virtuais e os arquivos de paginação são obrigatórios.
+
+Os nós do Windows não sobrecarregam a memória para os processos. O efeito líquido 
+é que o Windows não atingirá as condições de falta de memória 
+da mesma forma que o Linux, e processará a página em disco em vez de estar 
+sujeito ao encerramento por falta de memória (OOM). Se a memória for 
+superprovisionada e toda a memória física estiver esgotada, a paginação poderá diminuir o desempenho.
+
+## Gerenciamento de CPU {#resource-management-cpu}
+
+O Windows pode limitar a quantidade de tempo de CPU alocado para diferentes processos, 
+mas não pode garantir uma quantidade mínima de tempo de CPU.
+
+No Windows, o kubelet oferece suporte a uma flag de linha de comando para definir a
+[prioridade do escalonador](https://docs.microsoft.com/windows/win32/procthread/scheduling-priorities) do processo kubelet:
+ `--windows-priorityclass`. Essa flag permite que o processo kubelet obtenha
+mais fatias de tempo de CPU quando comparado a outros processos em execução no host do Windows.
+Mais informações sobre os valores permitidos e os seus significados estão disponíveis em
+[Classes prioritárias do Windows](https://docs.microsoft.com/en-us/windows/win32/procthread/scheduling-priorities#priority-class).
+Para garantir que os Pods em execução não prejudiquem o kubelet de ciclos de CPU, defina essa flag como `ABOVE_NORMAL_PRIORITY_CLASS` ou acima.
+
+## Reserva de recursos {#resource-reservation}
+
+Para contabilizar a memória e a CPU usadas pelo sistema operacional, o tempo de execução do contêiner 
+e pelos processos de host do Kubernetes, como o kubelet, você pode (e deve) 
+reservar recursos de memória e CPU com o  `--kube-reserved` e/ou `--system-reserved` flags de kubelet.
+No Windows, esses valores são usados apenas para calcular o nó
+[alocável](/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable) de recursos.
```

Comment:
Some bits that need attention here:
* Use "agente de execução de contêiner" (standardized translation) for "container runtime".
* "as flags `--kube-reserved` e/ou `--system-reserved` do kubelet" is a better translation than "o `--kube-reserved` e/ou `--system-reserved` flags de kubelet".
* How about using "recursos alocáveis pelo nó" instead of "nó alocável de recursos"?